### PR TITLE
Prepare for v0.6.0

### PR DESCRIPTION
**Pull request by Yatekii**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,8 +21,9 @@ env_logger = "0.7.1"
 log = "0.4.0"
 lazy_static = "1.4.0"
 colored = "1.8.3"
-probe-rs = { version = "0.5.1" }
-gdb-server = { version = "0.5.0" }
+probe-rs = { version = "*" }
```
Comment (Tiwalun) [inline Cargo.toml]: Please use proper versions here.

File: src/main.rs [hunk 2]
```diff
@@ -70,11 +69,23 @@ struct Opt {
...
+        help = "Use this flag to set the log level. Default is `warning`. Possible choices are [error, warning, info, debug, trace]."
     )]
     log: Option<log::Level>,
-    #[structopt(name = "speed", long = "speed", help = "Protocol speed in kHz")]
+    #[structopt(name = "speed", long = "speed", help = "The protocol speed in kHz.")]
     speed: Option<u32>,
+    #[structopt(
+        name = "restore-unwritten",
+        long = "restore-unwritten",
+        help = "Enable this flag to restore all bytes erased in the sector erase but not overwritten by any page. This option is only available with progressbars enabled."
```
Comment (Tiwalun) [inline src/main.rs]: Why does this depend on progress bars?

Comment (Tiwalun) [reply src/main.rs]: And if this is necessary, we should check for it and show an error if `--restore-unwritten` and `--disable-progressbars` are specified together.

Comment (Yatekii) [reply src/main.rs]: This is not true anymore. Will adjust :)

**Pull request merged**